content/posts/2018-10-21-reactive-state.md

```diff
@@ -0,0 +1,353 @@
+---
+title: "Reactive State Mutations via CQRS"
+description: "Managing state, the safe way. Without event sourcing (for now)."
+date: 2018-10-21
+slug: reactive-state-mutations
+---
+
+State-state-state. It surrounds us. Think hard enough and everything around
+will become either a state or a state mutation.
+The current time is a state and each passing second is a state mutation.
```

Comment:
heh, true if universe is nondeterministic

Reply:
Not sure if it is the smartest thing I’ve heard today or the most depressing one.